x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/upgrade.md

```diff
@@ -114,82 +116,68 @@ Status: `in progress`. The current test plan matches [Rule Immutability/Customiz
 
 ### Functional requirements
 
-- User should be able to install prebuilt rules with and without previewing what exactly they would install (rule properties).
 - User should be able to upgrade prebuilt rules with and without previewing what updates they would apply (rule properties of target rule versions).
-- If user chooses to preview a prebuilt rule to be installed/upgraded, we currently show this preview in a flyout.
-- In the prebuilt rule preview a tab that doesn't have any sections should not be displayed and a section that doesn't have any properties also should not be displayed.
+- User should be able to upgrade prebuilt rules with and without customizations. Where the following fields support customizations
```

Comment:
> Where the following fields support customizations

- Is it important to list the fields that support customization in the test plan for the rule upgrade workflow?
- We have a test plan `rule_customization.md` where we don't list them, somehow.
- We have two files in `/test_plans/detection_response/prebuilt_rules/shared_assets` that seem to duplicate this information